libs/agno/agno/tools/github.py

```diff
@@ -19,14 +21,43 @@ def __init__(
         search_repositories: bool = True,
         list_repositories: bool = True,
         get_repository: bool = True,
-        list_pull_requests: bool = True,
+        list_pull_requests: bool = False,
         get_pull_request: bool = True,
-        get_pull_request_changes: bool = True,
+        get_pull_request_changes: bool = False,
         create_issue: bool = True,
-        create_repository: bool = True,
+        create_repository: bool = False,
         delete_repository: bool = False,
         get_repository_languages: bool = True,
-        list_branches: bool = True,
+        list_branches: bool = False,
+        get_pull_request_count: bool = True,
+        get_repository_stars: bool = True,
+        get_pulls_by_query: bool = True,
+        get_pull_request_comments: bool = False,
+        create_pull_request_comment: bool = False,
+        edit_pull_request_comment: bool = False,
+        get_pull_request_with_details: bool = False,
+        get_repository_with_stats: bool = True,
+        list_issues: bool = False,
+        get_issue: bool = True,
+        comment_on_issue: bool = False,
+        close_issue: bool = True,
+        reopen_issue: bool = True,
+        assign_issue: bool = True,
+        label_issue: bool = True,
```

Comment:
Most of these don't need to be true by default. Lets make only `search_repositories` true by default